pkg/kubernetes/listing.js

```diff
@@ -3,12 +3,135 @@ define([
     "base1/cockpit",
     "base1/angular",
     "base1/term",
+    "kubernetes/client",
     "kubernetes/app"
-], function($, cockpit, angular, Terminal) {
+], function($, cockpit, angular, Terminal, kubernetes) {
     'use strict';
 
     var _ = cockpit.gettext;
 
+    var k8client =  kubernetes.k8client();
+    var adjust_btn = $('#adjust-entity-button'); 
+    var adjust_entity_dlg = $('#adjust-entity-dialog');
+    
+    var delete_pod_dlg = $('#delete-pod-dialog');
+    var delete_pod_btn = $('#delete-pod-button');
+
+
+    adjust_entity_dlg.on('show.bs.modal', function(e) {
+        var rc = $(e.relatedTarget).attr("data-entity");
```

Comment:
Instead of attaching the entire JSON entity to DOM element, could you put the UID in a data-uid attribuite, and then use client.lookup() to get the relevent entity.